controller/comments.controller.js

```diff
@@ -0,0 +1,18 @@
+const { selectComments } = require('../model/comments.model')
+
+const getComments = (req, res, next) => {
+    const article_id = req.params.article_id;
+    selectComments(article_id)
+    .then((comments) => {
+        if (comments && comments.length > 0) {
```

Comment:
What happens if I publish a brand new article that no one has read yet. How many comments would that have? Is that an error? And why not?